decide-dialga/decide
# [feat](vis) M3 Functional Increase

New functionality sumary:
- User-friendly interface for visualizer view.
- Local access for Chart.js library
- First version for dark mode.

In order to test this functionality:
1. Create a new voting
2. Make some votes
3. Close voting and tally
4. Visit ..../visualizer/{votingId}
5. In the view: activate o deactivate dark mode and check de user-friendly interface.

**Works in conventionals browsers.**
![m3-functional-increase-vis](https://user-images.githubusercontent.com/36485209/71754817-30f4c900-2e88-11ea-8207-422cd9a87954.PNG)
